zipline/utils/run_algo.py

```diff
@@ -71,7 +71,8 @@ def _run(handle_data,
          print_algo,
          metrics_set,
          local_namespace,
-         environ):
+         environ,
+         data_frame_loaders):
```

Comment:
I don't think there is anything special about this being a dataframe loader, right?